.github/workflows/test.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ruby: [2.3, 2.4, 2.5, 2.6, 2.7, head]
+        ruby: [2.3, 2.4, 2.5, 2.6, 2.7, '3.0', 3.1, head]
```

Comment:
'3.0' needs to be a string else it is converted to a number and rbenv install 3 will install latest 3.x version